content-fragment-export-to-target/src/aem-cf-console-admin-1/web-src/src/components/ExtensionRegistration.js

```diff
@@ -2,112 +2,122 @@
+                                        const hasUnpublished = typeof selections.find(cf => cf.status !== "published") !== "undefined";
+                                        if (!hasUnpublished) {
+                                            guestConnection.host.progressCircle.start();
+                                            try {
+                                                const auth = await guestConnection.sharedContext.get("auth");
+                                                const token = auth.imsToken;
+                                                const imsOrg = auth.imsOrg;
+                                                const repo = await guestConnection.sharedContext.get("aemHost");
 
-                      const exported = await triggerExportToAdobeTarget(
-                        token,
-                        repo,
-                        imsOrg,
-                        selections.map(cf => cf.id)
-                      );
-                      guestConnection.host.progressCircle.stop()
-                      if (exported) {
-                        guestConnection.host.toaster.display({
-                            variant: "positive",
-                            message: "Selected content fragments have been successfully synced with Adobe Target.",
-                        })
-                      } else {
-                        guestConnection.host.toaster.display({
-                          variant: "negative",
-                          message: 'Sync with Adobe Target failed. Please make sure CF offer is not being used in activity!'
-                        })
-                      }
-                    } catch (e) {
-                      console.log(e);
-                      guestConnection.host.progressCircle.stop()
-                      guestConnection.host.toaster.display({
-                        variant: "negative",
-                        message: 'Sync with Adobe Target failed.'
-                      })
-                    }
-                    return;
-                  }
+                                                const exported = await triggerExportToAdobeTarget(
+                                                    token,
+                                                    repo,
+                                                    imsOrg,
+                                                    selections.map(cf => cf.id)
+                                                );
+                                                guestConnection.host.progressCircle.stop()
+                                                if (exported) {
+                                                    guestConnection.host.toaster.display({
+                                                        variant: "positive",
+                                                        message: "Selected content fragments have been successfully synced with Adobe Target.",
```

Comment:
I would change the wording here to something like "Content fragments are scheduled for sync...". This may help users understand why their CF is not in target yet.

Reply:
fixed